common/national_focus/CHN.txt

```diff
@@ -150,7 +150,7 @@ focus_tree = {
 			has_country_leader = { ruling_only = yes name = "Queen Chrysalis" }
 		}
 		ai_will_do = {
-			factor = 10
+			factor = 0
```

Comment:
Maybe change it to 1 to appease Scroup

And besides, 1 in 200 is terrible odds anyway